pontoon/teams/views.py

```diff
@@ -210,6 +214,46 @@ def request_projects(request, locale):
     return HttpResponse('ok')
 
 
+@login_required(redirect_field_name='', login_url='/403')
+@require_POST
+def request_locales(request):
+    """Request locales to be added to pontoon."""
+    form = LocaleRequestForm(request.POST)
+    locales = Locale.objects.all().values_list('code', flat=True)
+
+    if not form.is_valid():
+        return HttpResponseBadRequest(form.errors.as_json())
+
+    code = form.cleaned_data['code']
+    name = form.cleaned_data['name']
+
+    if code in locales:
+        return HttpResponseBadRequest('Bad Request: Duplicate Locale Code specified')
+
+    user = request.user
+
+    if settings.PROJECT_MANAGERS[0] != '':
+        EmailMessage(
+            subject=u'New locale request for {locale} ({code})'.format(
+                locale=name, code=code
+            ),
+            body=u'''
+            Please add the Locale {locale} ({code}) to Pontoon
+            Requested by {user}:
```

Comment:
We should keep this the same as when requesting a team:

`Requested by {user}, {user_role}:`